boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.dts

```diff
@@ -130,21 +154,68 @@
 	};
 };
 
+&pinctrl_i2c_scl0 {
+	drive-open-drain;
+	output-high;
+};
+
+&pinctrl_i2c_sda0 {
+	drive-open-drain;
+	output-high;
+};
+
 &i2c_smb_1 {
 	status = "okay";
 	label = "I2C1";
 	port_sel = <1>;
+	pinctrl-0 = <&pinctrl_i2c_scl1 &pinctrl_i2c_sda1>;
+	pinctrl-names = "default";
+};
+
+&pinctrl_i2c_scl1 {
+	drive-open-drain;
+	output-high;
+};
+
+&pinctrl_i2c_sda1 {
+	drive-open-drain;
+	output-high;
 };
 
 &i2c_smb_2 {
 	status = "okay";
 	label = "I2C7";
 	port_sel = <7>;
+	pinctrl-0 = <&pinctrl_i2c_scl7 &pinctrl_i2c_sda7>;
+	pinctrl-names = "default";
+};
+
+&pinctrl_i2c_scl2 {
+	drive-open-drain;
+	output-high;
+};
+
+&pinctrl_i2c_sda2 {
+	drive-open-drain;
+	output-high;
 };
 
 &spi0 {
 	status = "okay";
 	clock-frequency = <4000000>;
 	lines = <4>;
 	chip-select = <0>;
+	port-sel = <0>; /* Shared SPI */
+
+	pinctrl-0 = < &pinctrl_qshd_cs0
+		      &pinctrl_qshd_clk
+		      &pinctrl_qshd_io0
+		      &pinctrl_qshd_io1
+		      &pinctrl_qshd_io2
+		      &pinctrl_qshd_io3 >;
+	pinctrl-names = "default";
+};
+
+&pinctrl_qshd_cs0 {
```

Comment:
@jvasanth1 per my debug this seems to cause the glitch in GPIO055 (RSMRST) in MAF mode. 
This was not part of MEC152x pinmux. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mec15xxevb_assy6853/pinmux.c#L428

Could you clarify this addition?

Reply:
This was added to configure the pins for shared spi application to test on the EVB board; 
but we see the problem now in MAF mode since this pin is used as RSMRST. 

I will remove this and then application can configure it in their .dts file if required.